### PR TITLE
Changed "show bulk update actions" label in Table (#4496)

### PR DESCRIPTION
**Pull request by 3t8**
Changed the label from `Show bulk update actions` to `Show update buttons` in Table component
#4496

### CODE REVIEW

File: docs/docs/widgets/table.md [hunk 1]
```diff
@@ -129,7 +129,7 @@ Show or hide download button at the Table footer.
 
 Show or hide filter button at the Table footer.
 
-### Show bulk update actions
+### Show update buttons
 
 It's enabled by default. Toggle `off` hide bulk update actions.
```
Comment (withshubh) [inline docs/docs/widgets/table.md:134]: Please update this too

Comment (withshubh) [inline docs/docs/widgets/table.md:134]: Please update this too

**Review by withshubh — CHANGES_REQUESTED** [review]

File: docs/docs/widgets/table.md (hunk 1 above)
Comment (3t8) [reply docs/docs/widgets/table.md]: Done

**Review by withshubh — APPROVED** [review]
Docs ✅

**Comment by 3t8** [comment]
Resolved conflicts and updated translation @withshubh

**Review by arpitnath — APPROVED** [review]
LGTM!

**Review by sherfin94 — APPROVED** [review]

**Review by sherfin94 — CHANGES_REQUESTED** [review]
Hey @3t8 , please resolve the conflicts.

**Pull request merged**